html/browsers/browsing-the-web/scroll-to-fragid/scroll-frag-non-utf8-encoded-document.html

```diff
@@ -0,0 +1,19 @@
+<!doctype html>
+<title>Fragment Navigation: fragment id should not be found in non UTF8 document</title>
+<meta name=timeout content=long>
+<meta http-equiv="Content-Type" content="text/html; charset=gbk"/>
+<script src="/resources/testharness.js"></script>
+<script src="/resources/testharnessreport.js"></script>
+<body>
+<div></div>
+<div id="&#x586f" style="position:absolute; top:100px;"></div>
+<div style="height:200em;"></div>
+<script>
+setup({ single_test: true });
+assert_equals(location.hash, "", "Page must be loaded with no hash");
+location.hash = '%89g';
+setTimeout(() => {
```

Comment:
I'm surprised this passes the lint check. Using an async_test wrapper and t.step_timeout seems slightly preferable?

Reply:
Not sure if a lint checker can catch this. Anyway, fixed.

Reply:
As far as I know unless this directory is in lint.ignore (I guess it might be given your other comment) the literal string setTimeout will trip it.

Reply:
It's because `lint.ignore` has `SET TIMEOUT: html/browsers/browsing-the-web/scroll-to-fragid/*`.